app/src/main/java/by/zharikov/myapplication/homework3/Homework3.kt

```diff
@@ -0,0 +1,156 @@
+package by.zharikov.myapplication.homework3
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import by.zharikov.myapplication.R
+import kotlinx.android.synthetic.main.activity_homework3.*
+
+class Homework3 : AppCompatActivity() {
```

Comment:
Too many repeated actions.Think about using collections and loops

Reply:
Updated